Libs/Python/BiochemPy/Compounds.py

```diff
@@ -107,8 +107,7 @@ def loadStructures(self,sources_array=[],db_array=[]):
         
     @staticmethod
     def parseFormula(formula):
-        if (
-                        formula is None or formula == "" or "noFormula" in formula or "null" in formula):
+        if (formula is None or formula == "" or "noFormula" in formula or "null" in formula):
```

Comment:
could achieve much the same with `if formula.strip() is in {None, "", "noFormula", "null"}:` assuming it is ok to match noFormula or null exactly.

Reply:
if (formula.strip() is in {None, "", "noFormula", "null"}):
                            ^
SyntaxError: invalid syntax